### PR TITLE
Bump version to 0.0.6 and update release notes

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

**Comment by Xiangs18** [comment]
Not quite sure why release notes are on README as well and they are not consistent.

**Comment by MrCreosote** [comment]
go ahead and merge the release notes from the readme into the actual release notes and delete them from the readme

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
+### Version  0.0.6
+- Updated Dockerfile and upgraded kb-sdk with patch in GHA
```
Comment (MrCreosote) [inline RELEASE_NOTES.md]: Can you add a few words about why you updated the dockerfile

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
+### Version  0.0.6
+- Updated the Dockerfile so that the updated Docker daemon in GHA will pull the image
```
Comment (MrCreosote) [inline RELEASE_NOTES.md]:
```suggestion
- Updated the Dockerfile base image and build commands to fix the failing build
```

Comment (Xiangs18) [reply RELEASE_NOTES.md]: How about `Updated the base image and build commands in the Dockerfile to fix the failing build.`?

Comment (MrCreosote) [reply RELEASE_NOTES.md]: sure

Comment (Xiangs18) [reply RELEASE_NOTES.md]: 👍

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**